cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -513,6 +513,9 @@ rules:
 - apiGroups: [""]
   resources: ["configmaps"]
   verbs: [ "get", "list", "watch", "create", "update", "delete" ]
+- apiGroups: ["ceph.rook.io"]
+  resources: ["cephclusters", "cephclusters/finalizers"]
+  verbs: ["*"]
```

Comment:
Does the OSD really need get, create, edit, delete access to the cephclusters CRD or just the cephclusters/finalizer?

Reply:
Good question, i'll narrow this down more.

Reply:
I confirmed the failure was hit unless we request both `cephclusters and `cephclusters/finalizers`. The only verb that we don't need is `watch`. All the others were required in my testing.

Reply:
Ah well it was worth a try.